setup.py

```diff
@@ -18,7 +17,7 @@
     ]
 }
 DESCRIPTION = (
-    'renders pyecharts as image' +
+    'save pyecharts output file as image' +
```

Comment:
'render' is better than 'save' - 'save' is put everything as it is to a file, whereas 'render' would transform something and then save.

plus: this is not the place to update, please read the first line of the setup.py.  This package uses moban and description can be found in pyecharts-snapshot.yml